test/apiv2/10-images.at

```diff
@@ -440,4 +440,18 @@ t GET "libpod/events?stream=false&since=$START"  200  \
   .Actor.Attributes.name="localhost:5000/idonotexist" \
   .Actor.Attributes.error~".*connection refused"
 
+# test empty RepoTags and RepoDigests is an empty array
+IIDFILE=$(mktemp)
+podman image build --iidfile $IIDFILE -<<EOF
+from alpine
+RUN :
+EOF
+
+t GET images/json 200 \
+  .[2].RepoTags=[] \
```

Comment:
I still think this also should check `.[2].Id=$(< $IIDFILE)` to ensure you are actually checking the right image